src/AntiPlagiarism/AntiPlagiarism.Vsix/Resources/VSIXResource.resx

```diff
@@ -117,7 +117,25 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="AcumaticaSourcesWorkModeDescription" xml:space="preserve">
```

Comment:
Do we send UI strings to review in this project? Probably we should.

Reply:
No we do not send them becuase there are too many strings in the UI. Overall there is a huge amount of strings in the UI. The collection of all of them into the separate resource file is quite tedious job which imho should be done some other time in scope of another item

Reply:
@metallmonster not to mention that this will greatly slow down the development as the doc team probably will need to install the extension to get into the context in which the string is used. And this is a low priority project for them therefore the verifcation can take a lot of time. I'm not sure that we need to verify UI strings with the doc team but if we're going to do that then it's better to do it in a separate task for all UI.